src/uikit/Multicard/index.js

```diff
@@ -90,31 +84,38 @@ class Multicard extends Component {
         ) : (
           <CardWrapper className={className} inactive={inactive}>
             <HeaderWrapper inactive={inactive}>
-              {activeTab.headerComponent ? (
-                activeTab.headerComponent(childProps)
+              {activeTab && activeTab.headerComponent ? (
+                activeTab.headerComponent(this.childProps)
               ) : (
-                <CardHeader title={title} badge={badgeNumber}>
-                  {!inactive &&
-                    tabs.map((tab, i) => (
-                      <TabMenu
-                        key={i}
-                        active={i === contentIndex}
-                        onClick={() => {
-                          this.setIndex(i);
-                          trackUserInteraction({
-                            category: TRACKING_EVENTS.categories.user.dashboard.widgets._multiCard,
-                            action: `Tab: ${TRACKING_EVENTS.actions.click}`,
-                            label: JSON.stringify({ card: title, tab: tab.nav }),
-                          });
-                        }}
-                        title={tab.nav}
-                      />
-                    ))}
-                </CardHeader>
+                <CardHeader title={title} />
               )}
+              <Row>
+                {!inactive &&
+                  tabs.map((tab, i) => (
+                    <TabMenu
+                      key={i}
+                      active={i === currentTabIndex}
+                      onClick={() => {
+                        this.setIndex(i);
+                        trackUserInteraction({
+                          category: TRACKING_EVENTS.categories.user.dashboard.widgets._multiCard,
```

Comment:
@bdolly moved `tab` tracking code to here because the headers are now passed in as components